src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -79,24 +86,29 @@ export class HealthComponent implements OnInit, OnDestroy {
       100 * (data.df.stats.total_used_raw_bytes / data.df.stats.total_bytes)
     );
 
+    chart.dataset[0].label = 'asd';
```

Comment:
What does 'asd' means?

Reply:
It was just some code I added to tests the labels and forgot to remove.
It has been removed, thanks for spotting.